.github/workflows/pr-meta.yaml

```
@@ -0,0 +1,23 @@
+name: Pull Request Meta
+
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```

It won't work with just pull_request, the build won't have permission for the PR modification. 

pull_request_target must be used, it is also in the README: https://github.com/toshimaru/auto-author-assign?tab=readme-ov-file#usage

Related discussion here: https://github.com/toshimaru/auto-author-assign/issues/22

---

Fixed.